src/auth/index.js

```diff
@@ -264,6 +268,7 @@ export default class Auth {
       return this._get();
     }).then((auth) => {
       newUserAuth = Object.assign(auth, newUserAuth);
```

Comment:
[minor] Seems like there are many other methods here that could be made synchronous since we can load data for refresh, access token, and so on up front. The only time we would need to be async would be for initialization, making a network request, and setting new data. We don't strictly need to do this for everything, but we could. Piecemeal is fine as well. The Android SDK does the same right now but could load more up front.